src/components/Configure/state/HydratedRevisionContext.tsx

```diff
@@ -117,9 +125,25 @@ export function HydratedRevisionProvider({
   if (isError(ErrorBoundary.HYDRATED_REVISION, errorIntegrationIdentifier)) {
     const intNameOrId = integrationObj?.name || integrationId || 'unknown integration';
     const errorMsg = `Error retrieving integration details for '${intNameOrId
-    }. This is sometimes caused by insufficient permissions with your credentials'`;
-
-    return <ComponentContainerError message={errorMsg} />;
+    }. This is sometimes caused by insufficient permissions with your credentials. ' 
+    ${readableErrorMsg ? `: ${readableErrorMsg}` : ''}`;
```

Comment:
Let's get rid of "This is sometimes caused by insufficient permissions with your credentials." that was a generic message, and was causing a lot of confusion, since it's usually not actually a permission issue.